### PR TITLE
Fixed warnings to increase code quality

### PR DESCRIPTION
**Pull request by AliceBilc**

### CODE REVIEW

File: services/ppac/src/test/java/app/coronawarn/datadonation/services/ppac/android/testdata/JwsGenerationUtil.java [hunk 1]
```diff
@@ -47,9 +47,8 @@ public static JsonWebSignature createJsonWebSignature(Map<String, Serializable>
       int secondDot = signedJWSString.indexOf('.', firstDot + 1);
       byte[] signatureBytes = Base64.decodeBase64(signedJWSString.substring(secondDot + 1));
       byte[] signedContentBytes = StringUtils.getBytesUtf8(signedJWSString.substring(0, secondDot));
-      JsonWebSignature signature =
-          new JsonWebSignature(header, payload, signatureBytes, signedContentBytes);
-      return signature;
+      return new JsonWebSignature(header, payload, signatureBytes,
+          signedContentBytes); //Local variable 'signature' is redundant
```
Comment (hilmarf) [inline services/ppac/src/test/java/app/coronawarn/datadonation/services/ppac/android/testdata/JwsGenerationUtil.java:51]:
```suggestion
          signedContentBytes);
```

File: services/ppac/src/test/java/app/coronawarn/datadonation/services/ppac/android/testdata/JwsGenerationUtil.java [hunk 2]
```diff
@@ -59,33 +58,34 @@ public static String createCompactSerializedJws(Map<String, Serializable> payloa
...
     }
     return null;
   }
-  
+
   private static PrivateKey getPrivateKey() {
     try {
       URL url = JwsGenerationUtil.class.getResource("/certificates/test.key");
-      PEMParser pemParser = new PEMParser(new FileReader(new File(url.getPath())));
+      PEMParser pemParser = new PEMParser(new FileReader((url.getPath()))); // local variable new File is redundant
```
Comment (hilmarf) [inline services/ppac/src/test/java/app/coronawarn/datadonation/services/ppac/android/testdata/JwsGenerationUtil.java:88]:
```suggestion
      PEMParser pemParser = new PEMParser(new FileReader(url.getPath()));
```

**Review by ioangut — APPROVED** [review]

File: services/ppac/src/test/java/app/coronawarn/datadonation/services/ppac/android/testdata/JwsGenerationUtil.java (hunk 1 above)
Comment (AliceBilc) [reply services/ppac/src/test/java/app/coronawarn/datadonation/services/ppac/android/testdata/JwsGenerationUtil.java]: Deleted the comment

File: services/ppac/src/test/java/app/coronawarn/datadonation/services/ppac/android/testdata/JwsGenerationUtil.java (hunk 2 above)
Comment (AliceBilc) [reply services/ppac/src/test/java/app/coronawarn/datadonation/services/ppac/android/testdata/JwsGenerationUtil.java]: Deleted the comment

**Pull request merged**